orchestrator/crates/settlement-clients/starknet/src/lib.rs

```diff
@@ -136,13 +138,16 @@ impl SettlementClient for StarknetSettlementClient {
             .update_state(snos_output, layout_bridge_output)
             .await
             .map_err(|e| eyre!("Failed to update state with calldata: {:?}", e))?;
-        tracing::info!(
+
+        let tx_hash = invoke_result.transaction_hash.to_hex_string();
+        info!(
             log_type = "completed",
             category = "update_state",
             function_type = "calldata",
-            "State updated with calldata."
+            tx_hash = %tx_hash,
+            "State update transaction submitted to Starknet with calldata"
```

Comment:
calldata update can happen without starknet being l2 tho
```suggestion
            "State update transaction submitted with calldata"
```

Reply:
why? this is a starknet client no?